src/utils/normalization.ts

```diff
@@ -107,3 +107,31 @@ export const normalize = (payload: OperationsRepository) => {
 
   return { apis, documentation } as OperationsRepository;
 };
+
+export const emptyObject = (object: any, preserveValueKeys: string[], ignoreNestedKeys: string[]) => {
+  for (const key in object) {
+    if (typeof object[key] === 'object' && !ignoreNestedKeys.includes(key)) {
+      emptyObject(object[key], preserveValueKeys, ignoreNestedKeys);
+    } else {
+      // eslint-disable-next-line functional/immutable-data
+      object[key] = preserveValueKeys.includes(key) ? object[key] : emptyReturn(object[key]);
+    }
+  }
+  return object;
+};
+
+const emptyReturn = (value: any) => {
```

Comment:
The only big problem with this approach that I can see is we land up with files that have literally empty fields in them. It means the user doesn't have an easy reference as to what data should be in a file/field. They always have to refer to something else, which is slow and painful. 

```
{
  "name": "",
  "templateId": "",
  "endpointId": "",
  "parameters": "",
  "decodedParameters": {}
}
```

Reply:
what do you think would be better? do we populate the templates with mockdata maybe from the coingecko api ?